ebuild-writing/functions/src_configure/configuring/text.xml

```diff
@@ -74,5 +74,56 @@ form can be used.
 </p>
 </body>
 
+<section>
+<title><c>econf</c> Options</title>
+<body>
+
+<p>
+<c>econf</c> is designed to work with configure scripts generated by
+GNU Autoconf. It first passes the default options listed below to the configure
+script, followed by any additional parameters passed to <c>econf</c>.
+</p>
+
+<ul>
+  <li><c>--prefix="${EPREFIX}"/usr</c></li>
+  <li><c>--mandir="${EPREFIX}"/usr/share/man</c></li>
+  <li><c>--infodir="${EPREFIX}"/usr/share/info</c></li>
+  <li><c>--datadir="${EPREFIX}"/usr/share</c></li>
+  <li><c>--sysconfdir="${EPREFIX}"/etc</c></li>
+  <li><c>--localstatedir="${EPREFIX}"/var/lib</c></li>
+  <li>
+    <c>--build="${CBUILD}"</c> (only passed if <c>CBUILD</c> is non-empty)
+  </li>
+  <li><c>--host="${CHOST}"</c></li>
+  <li>
+    <c>--target="${CTARGET}"</c> (only passed if <c>CTARGET</c> is non-empty)
+  </li>
+  <li>
+    <c>--libdir</c> is set from the value of the <c>LIBDIR_${ABI}</c> variable
+    in profiles.
+  </li>
+  <li><c>--disable-dependency-tracking</c></li>
+  <li><c>--disable-silent-rules</c></li>
+</ul>
+
+<p>
+In EAPI 6 and later, the following options are passed in addition:
```

Comment:
I'm not sure if 'in addition' doesn't require the subject.

Reply:
> I'm not sure if 'in addition' doesn't require the subject.

I don't think so: https://www.macmillandictionary.com/dictionary/british/in-addition